package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -26,16 +37,27 @@ def _is_json_serializable(obj: Any):
 class SQLiteStore(BaseSessionStore):
     """Stores the session data on the sqlite db."""
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, remote_path: Optional[str] = None, **kwargs):
+        """Initializes the SQLiteStore object."""
         super().__init__(*args, **kwargs)
         self._db_session_class = make_db_session_factory(self.location)
```

Comment:
Not your code, ,but this seems to be session object and not session class. Could you rename it to simply `_db_session`?

Reply:
This is actually a class: https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.sessionmaker. The docs here aren't very clear because they call the returned `Session` an object, but if you read more it is definitely a class.
```
Session = sessionmaker(engine)

with Session() as session:
```